test/integration/console.cc

```diff
@@ -37,7 +37,7 @@ TEST(Console_TEST, LogInitAfterConsoleOut)
   gz::common::Console::SetVerbosity(4);
 
   // Before console is initialized, logs go to default file location
-  gzerr << "Error before logging initialized" << std::endl;
+  gzlog << "Error before logging initialized" << std::endl;
```

Comment:
Does `gzerr` not log to the default file location?

Reply:
Reverted in https://github.com/gazebosim/gz-common/pull/615/commits/f3d168cfdc7eb4a8575edc4dfbf24cd8a98a5c52

Reply:
gzerr restored in [f3d168c](https://github.com/gazebosim/gz-common/pull/615/commits/f3d168cfdc7eb4a8575edc4dfbf24cd8a98a5c52).